google/cloud/spanner_v1/client.py

```diff
@@ -313,6 +314,9 @@ def instance(
         :param node_count: (Optional) The number of nodes in the instance's
                             cluster; used to set up the instance's cluster.
 
+        :type labels: str
```

Comment:
In Node.js, the type is `labels?: {[k: string]: string} | null`. It seems that below you expect it to be a dict. Should it be made explicit here? Otherwise, if it's just a string, then it should be `label`.

Reply:
Whoops! Good catch! I'll update it